AUTHORS.md

```diff
@@ -0,0 +1,14 @@
+# LocalAdmin
```

Comment:
This file doesn't make sense, source code doesn't need to include the contributors.

Reply:
Deleted.